src/Tizen.NUI/src/public/Position.cs

```diff
@@ -521,10 +521,13 @@ public static Position Zero
         /// The x component.
         /// </summary>
         /// <since_tizen> 3 </since_tizen>
+        [Obsolete("Please do not use this setter, Deprecated in API8, will be removed in API10. please use new Position(...) constructor")]
```

Comment:
```suggestion
            [Obsolete("Do not use this setter, which is deprecated in API8 and will be removed in API10. Use the new Position(...) constructor")]
```

Reply:
Thank you for review.
these are fixed by the following patches.

https://github.com/Samsung/TizenFX/pull/4356
https://github.com/Samsung/TizenFX/pull/4357

thank you.